lms/djangoapps/teams/csv.py

```diff
@@ -170,22 +170,34 @@ def import_succeeded(self):
 
     def set_team_membership_from_csv(self, input_file):
         """
-        Assigns team membership based on the content of an uploaded CSV file.
+        Wrapper to parse CSV file to use for setting team memberships.
```

Comment:
something about this line rubs me the wrong way. I think the term "wrapper" by itself.
Maybe change it to "Wraps `set_team_memberships`, parsing input file and passing in a DictReader?"

Reply:
Updated docstring